craft_application/commands/lifecycle.py

```diff
@@ -471,32 +471,23 @@ def _run(
             # Run the rest of this outside the managed instance.
             return
 
-        emit.progress("Testing project.")
+        emit.progress("Testing project")
 
         dest = pathlib.Path.cwd() / TEMP_SPREAD_FILE_NAME
```

Comment:
is this variable still necessary? It doesn't seem to be written to anymore

Reply:
This is a leftover from a previous implementation and shouldn't be necessary anymore.